plugins/modules/public_cloud_block_storage.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: public_cloud_block_storage
+
+short_description: Manage OVH API for public cloud volume.
+
+description:
+    - This module manage, volume of an instance on OVH public Cloud.
```

Comment:
This module only manage volumes, not instance ?
I'm not sure, because you can attach a volume to an instance with `public_cloud_block_storage` AND `public_cloud_block_storage_instance` :thinking:

Reply:
The `public_cloud_block_storage` module create or delete a block storage, that all.

The `public_cloud_block_storage_instance` module, allow to attach or detach a block storage to an instance.

Reply:
Ok, I missread the parameter, I though the "image_id" were and "instance id" to attach to :)

Can you please rephrase btw ?  `This module manage, volume of an instance on OVH public Cloud.` maybe `This module manage volume creation/deletion on OVH public Cloud`

Reply:
It's done